src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,63 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Tagging {
+    private Person person;
+    private ReadOnlyPerson readPerson;
+    private Tag tag;
+    private String type;
+    private static final List<Tagging> TAGGINGS = new ArrayList<Tagging>();
```

Comment:
Based on the class diagram given in the exercise, the list of `Tagging` objects should not be in this class.